nautilus_trader/persistence/catalog/parquet.py

```diff
@@ -325,14 +325,6 @@ def _write_chunk(
         directory = self._make_path(data_cls=data_cls, identifier=identifier)
         self.fs.mkdirs(directory, exist_ok=True)
 
-        if isinstance(data[0], Instrument):
```

Comment:
Removing this is correct, as we _do_ want to be able to write multiple versions of an instrument - as they can change over time.